src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs

```diff
@@ -369,5 +373,62 @@ private void ScrollViewer_PreviewMouseWheel(object sender, MouseWheelEventArgs e
             }
             e.Handled = true;
         }
+
+        private void importTextBlock_MouseLeftButtonDown(object sender, MouseButtonEventArgs e)
+        {
+            string[] fileFilter = { string.Format(Res.FileDialogImportSettingsFiles, "*.xml") };
+            System.Windows.Forms.OpenFileDialog openFileDialog = new System.Windows.Forms.OpenFileDialog();
```

Comment:
There is already a MessageBoxService (inside the MessageBoxUtilities.cs) that helps to show message boxes then you could use that class.

Reply:
Yes I use it when I handling the exception messages, In the highlighted case it needs to use the openFileDialog to grab a file as other places Does, thanks for the recommendation.